cclib/parser/data.py

```diff
@@ -180,6 +191,7 @@ class ccData:
        "vibramans":        Attribute(numpy.ndarray,    'raman',                       'vibrations:intensities'),
        "vibrmasses":       Attribute(numpy.ndarray,    'reduced masses',              'vibrations'),
        "vibsyms":          Attribute(list,             'vibration symmetry',          'vibrations'),
+       "WBO_matrix":       Attribute(numpy.ndarray,    'WBO matrix',                  'properties:NBO'),

```

Comment:
```suggestion
       "bondorders":       Attribute(numpy.ndarray,    'bond orders',                  'properties'),

```
Might want to adjust the cjson name but this will do for now :)

Reply:
sounds good!